### PR TITLE
test: init e2e test

### PR DESCRIPTION
**Pull request by duyhungtnn**
This PR init a set of e2e tests for checking the `get_evaluation` feature

Screenshot 2024-05-23 at 11 11 56

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -56,6 +56,7 @@ e2e: copy-genfiles
 	rm -rf $(CURDIR)/__e2e
 	$(NPM_BIN_DIR)/babel e2e --extensions '.ts' --config-file "$(CURDIR)/babel-e2e.config.js" --out-dir "__e2e/__test__"
 	$(NPM_BIN_DIR)/babel lib --extensions '.mjs' --config-file "$(CURDIR)/babel-e2e.config.js" --out-dir "__e2e/lib"
+	cp package.json __e2e
```
Comment (duyhungtnn) [inline Makefile:59]: on the code, we are reading the lib `version` from the `package.json`

File: e2e/constants/constants.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+export const USER_ID = 'bucketeer-js-user-id-1'
+
+export const FEATURE_ID_BOOLEAN = 'feature-js-e2e-boolean'
+export const FEATURE_ID_STRING = 'feature-js-e2e-string'
+export const FEATURE_ID_INT = 'feature-js-e2e-int'
+export const FEATURE_ID_DOUBLE = 'feature-js-e2e-double'
+export const FEATURE_ID_JSON = 'feature-js-e2e-json'
+
+export const GOAL_ID = 'goal-js-e2e-1'
+export const GOAL_VALUE = 1.0
```
Comment (duyhungtnn) [inline e2e/constants/constants.ts:13]: I'd like to replace the test data sample here with a set for the node.js SDK

Comment (cre8ivejp) [reply e2e/constants/constants.ts:13]: It will be better to create new flags for nodejs tests.

Comment (duyhungtnn) [reply e2e/constants/constants.ts:13]: I updated the nodejs flags

**Review by cre8ivejp — APPROVED** [review]
Than you!

**Pull request merged**